app/views/two_factor_authentication/otp_expired/show.html.erb

```diff
@@ -31,5 +31,13 @@
           text: t('links.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
-      ],
+        {
+          url: @authentication_options_path,
+          text: t('two_factor_authentication.login_options_link_text'),
+        },
+        @unconfirmed_phone && {
```

Comment:
We could probably get rid of the extra instance variable and just use `@use_another_phone_path`, which would be `nil` in the cases where we don't want to show the option.

```suggestion
        @use_another_phone_path && {
```